src/features/room/create/ui/RoomFormModal.module.css

```
@@ -1,363 +1,318 @@
+  bottom: 130px;
   display: flex;
   align-items: center;
   justify-content: center;
   padding: 24px;
-  background: rgba(0, 0, 0, 0.48);
 }
 
 .modal {
+  --panel-horizontal-padding: 16px;
+  --panel-content-width: 1060px;
   position: relative;
```

Stylelint (17.11.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)